python/lsst/drp/tasks/gbdesAstrometricFit.py

```diff
@@ -302,6 +314,14 @@ class GbdesAstrometricFitConfig(
         doc="Set the random seed for selecting data points to reserve from the fit for validation.",
         default=1234,
     )
+    saveModelParams = pexConfig.Field(
+        dtype=bool,
+        doc=(
+            "Save the parameters and covariance of the WCS model. Default to "
+            "false because this can be very large."
```

Comment:
I don't know if these needs to be added to the docstring necessarily but how many parameters is typical? Or is the possible range very large?

Reply:
The range is really large because the number of model coefficients depends on the polynomial order for the two mappings, the number of detectors, and the number of visits. For a tract of HSC RC2, it's ~5000 parameters.